api/variables_test.go

```diff
@@ -307,3 +307,67 @@ func TestVariable_CreateReturnsContent(t *testing.T) {
 	must.NotNil(t, sv1n)
 	must.Eq(t, sv1.Items, sv1n.Items)
 }
+
+func TestVariables_LockRenewRelease(t *testing.T) {
```

Comment:
looks like this tests only happy paths -- are there any error conditions that we want to test here?  or are we considering that adequately covered by tests on the server side?

relatedly, do we want to include any api-client-side validation for things like "TTL is too small" or any others?  I'm not sure how common that is in our api package.

Reply:
According to what I have seen, the API is a thin wrapper, no validations or defaults